### ISSUE TITLE
API documentation

### ISSUE THREAD

**Issue by crfrolik** [aboutcode-org/debian-inspector#47]
I see there is a `docs` folder, but I have no idea how to use this.
Is there a place to see API documentation or examples online?

**Comment by Gerardwx** [comment]
Agree. The README is all about Orign and Why but there's nothing as to why anyone would install this package in the first place.
